Dockerfile.base

```diff
@@ -55,12 +54,14 @@ RUN echo en_US.UTF-8 UTF-8 >> /etc/locale.gen \
   && distro=$(lsb_release -is | awk '{print tolower($0)}') \
   # Determine the Distro version (bullseye, xenial, etc)
   # Note: sid is aliased to bullseye, because Docker doesn't have a matching apt repo
-  && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys ${DOCKER_KEY} \
-  && curl -fsSL https://download.docker.com/linux/${distro}/gpg | apt-key add - \
+  && mkdir -p /etc/apt/keyrings \
+  && ( curl -fsSL https://download.docker.com/linux/${distro}/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg ) \
   && version=$(lsb_release -cs | awk '{gsub("sid", "bullseye"); print $0}') \
-  && ( add-apt-repository "deb [arch=$(dpkg --print-architecture)] https://download.docker.com/linux/${distro} ${version} stable" ) \
+  && ( echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/${distro} ${version} stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null ) \
   && apt-get update \
   && apt-get install -y docker-ce docker-ce-cli containerd.io --no-install-recommends --allow-unauthenticated \
-  && [[ $(lscpu -J | jq -r '.lscpu[] | select(.field == "Vendor ID:") | .data') == "ARM" ]] && echo "Not installing docker-compose. See https://github.com/docker/compose/issues/6831" || ( curl -sL "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose ) \
+  && ( [[ $(lscpu -J | jq -r '.lscpu[] | select(.field == "Vendor ID:") | .data') == "ARM" ]] && echo "Not installing docker-compose. See https://github.com/docker/compose/issues/6831" || ( curl -sL "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose ) ) \
+  && ( [[ $(lsb_release --codename | cut -f2) == "focal" ]] && ( echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ /" | tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/Release.key" | apt-key add - && apt-get update) || : ) \
```

Comment:
will this work for debian and non-jammy ubuntu forks?

Not sure we should make it _only_ focal unless we have to

Reply:
Kubic repro is discontinued as described [here](https://podman.io/getting-started/installation) and Focal is the latest LTS where podman was introduced... So It seems that is not right to try to add podman to bionic builds.
Also, debian sid and Bullseye doesn't require kubic repro... So that's why I added this only for focal.

Reply:
We just had an issue with the kubic repo being down and blocking us from running `apt-get update`. Is there a reason we need `podman` installed inside the containers?

Reply:
I use this Github container to build oci images with buildah actions and other redhat actions. 
Without podman inside the container, you cannot execute those actions...
Also, if you are blocked from kubic repo, it should be related to the host machine and not the container.

Reply:
Honestly the best path here is to remove any non-standard repos at the end of the build which would lower it to just canonical mirros being down. which would be a bigger issue than a few third parties